dsn_test.go

```diff
@@ -62,137 +147,15 @@ func TestDsnDeserializeInvalidJSON(t *testing.T) {
 	}
 }
 
-func TestValidDsnInsecure(t *testing.T) {
-	url := "http://username@domain:8888/42"
-	dsn, err := NewDsn(url)
-
-	if err != nil {
-		t.Error("expected dsn to be correctly created")
-	}
-	assertEqual(t, url, dsn.String())
-}
-
-func TestValidDsnNoPort(t *testing.T) {
-	url := "http://username@domain/42"
-	dsn, err := NewDsn(url)
-
-	if err != nil {
-		t.Error("expected dsn to be correctly created")
-	}
-	assertEqual(t, 80, dsn.port)
-	assertEqual(t, url, dsn.String())
-	assertEqual(t, "http://domain/api/42/store/", dsn.StoreAPIURL().String())
-}
-
-func TestValidDsnPrefixed(t *testing.T) {
-	url := "http://username@domain/prefixed/42"
-	dsn, err := NewDsn(url)
-
-	if err != nil {
-		t.Error("expected dsn to be correctly created")
-	}
-	assertEqual(t, "http://domain/prefixed/api/42/store/", dsn.StoreAPIURL().String())
-}
-
-func TestValidDsnInsecureNoPort(t *testing.T) {
```

Comment:
Now covered by "MinimalSecure" test case.
https://github.com/getsentry/sentry-go/pull/112/files#diff-eeb2e1d2b086ffdc5e8a6e60178702aaR33

Note that the name of the test here was misleading. It said "Insecure" but the URL was "https".